resource/closecaption_german.txt

```diff
@@ -709,82 +709,82 @@
 "[english]bastille.Selection3"		"<clr:205,127,50>Bastille: Medic reporting."
 "bastille.Selection4"		"<clr:205,127,50>Bastille: Melde mich zum Dienst."
 "[english]bastille.Selection4"		"<clr:205,127,50>Bastille: Reporting."
-"crash.SargeIdleReply0"		"<clr:238,180,34>Crash: Du weißt wirklich, wie man eine Gruppe in Sicherheit bringt, Sarge."
+"crash.SargeIdleReply0"		"<clr:238,180,34>Crash: Da fühl ich mich doch gleich viel sicherer, Sarge."
 //Crash
 "[english]crash.SargeIdleReply0"		"<clr:238,180,34>Crash: You really know how to make a squad feel safe, Sarge."
-"crash.SargeIdleReply1"		"<clr:238,180,34>Crash: Heißt das, du willst jetzt nach Hause gehen, Sarge?"
+"crash.SargeIdleReply1"		"<clr:238,180,34>Crash: Du wirst doch jetzt nicht etwa kneifen, Sarge?"
 "[english]crash.SargeIdleReply1"		"<clr:238,180,34>Crash: That mean you wanna go home now, Sarge?"
-"crash.SynUpSpotted0"		"<clr:238,180,34>Crash: Hey, schau mal, die haben SynUp Cola, mein Lieblingsgetränk."
+"crash.SynUpSpotted0"		"<clr:238,180,34>Crash: Schaut mal, die haben hier SynUp Cola, mein Lieblingsgetränk."
 "[english]crash.SynUpSpotted0"		"<clr:238,180,34>Crash: Hey look, they got SynUp Cola, my favorite."
-"crash.BigAlienDeadReply0"		"<clr:238,180,34>Crash: Erinnert mich an den letzten Commander."
+"crash.BigAlienDeadReply0"		"<clr:238,180,34>Crash: Erinnert mich an den letzten Kommandanten."
```

Comment:
@BenLubar Do you know the context of `crash.BigAlienDeadReply0`?
Original line: 'Crash: Reminds me of that last Commander.'

Does he see an already dead alien and thinks about a group before the current one which cleaned up?
Is he maybe talking about some dead guy they find after killing a big alien (his corpse maybe lying around somewhere)?
Has 'Commander' even in the same context as in [here](https://github.com/ReactiveDrop/reactivedrop_translations/discussions/202)?

Reply:
I think it's a meta-joke about missions failing due to the game being hard.

Reply:
@DukeDukeEm 
Abschließendes Urteil?
In den deutschen Sprachdateien sind beide Varianten (je nach Kontext) im Gebrauch; so wie ich es verstehe, bezieht sich Crash auf den Anführer seines vorigen Trupps, also wäre auch eine Bezeichnung wie "Trupp(en)führer" oder "Einsatzleiter" (wobei sich letzteres eigentlich ziemlich gut für die Stimme aus dem Off eignen würde…?) möglich.

Reply:
Bleiben wir bei Kommandanten, das passt.